lineman/app/models/discussion_reader_model.coffee

```diff
@@ -12,11 +12,13 @@ angular.module('loomioApp').factory 'DiscussionReaderModel', (BaseModel) ->
       @volume = null
       @lastReadSequenceId = -1
 
-      if data.discussion_id?
-        discussion = @recordStore.discussions.find(data.discussion_id)
-        _.extend data,
-          id:       discussion.id
-          group_id: discussion.groupId
+      # commented so I can ask gdpelican about it
```

Comment:
Won't this mean you can't call `discussion_readers.find(discussion_id)` and find the thing you're looking for?

Don't think I wrote this, but can't remember.